src/components/NcColorPicker/NcColorPicker.vue

```diff
@@ -325,7 +326,7 @@ function hexToRGB(hex: string): [number, number, number] {
 						v-model="currentColor"
 						class="color-picker__advanced"
 						:disable-alpha="true"
-						:disable-fields="!advancedFields"
+						:disable-fields="false"
```

Comment:
This basically always enables the advanced fields even if you pass `advancedFields = false` to the component